app/src/main/java/org/mifos/mobile/ui/activities/AboutUsActivity.kt

```diff
@@ -11,10 +12,13 @@ import org.mifos.mobile.ui.fragments.AboutUsFragment
  * On 11/03/19.
  */
 class AboutUsActivity : BaseActivity() {
+    private lateinit var binding : ActivityContainerBinding
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_container)
+        binding = ActivityContainerBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
         setToolbarTitle(getString(R.string.about_us))
```

Comment:
do check if the toolbar is visible in the dark mode without changing this line

Reply:
please checkout the video I attached in SettingActivity.kt